harvester/harvester-installer
# Bump Rancher 2.8.1 and RKE2 1.27.9

**Related Issue:**

https://github.com/harvester/harvester/issues/5041

**Test plan:**
- Start the cluster and make sure the cluster bootstrap.
- Verify the Rancher version and RKE2 version.
